### PR TITLE
Kevin da grat

### PR DESCRIPTION
**Pull request by kevintxwu**
60% done! Didn't break anything and added the bulk of basic cleaning. Fancy stuff and more cleaning by the next PR!!!! (Probably tmmro or weekend or something).

### CODE REVIEW

**Comment by chenhoward** [comment]
l o l r e k t

**Comment by kevintxwu** [comment]
MERGE PLS

File: db/schema.rb [hunk 1]
```diff
@@ -19,22 +19,22 @@
   create_table "admins", force: :cascade do |t|
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "first_name"
-    t.string   "last_name"
-    t.string   "email",                  default: "", null: false
-    t.string   "encrypted_password",     default: ""
-    t.string   "reset_password_token"
+    t.string   "first_name",             limit: 255
```
Comment (chenhoward) [inline db/schema.rb]: FUUUU I HATE WHEN THIS HAPPENS

File: app/assets/stylesheets/pages/home.scss [hunk 2]
```diff
@@ -1,22 +1,61 @@
...
+}
+
+.home-button-spacer {
+  @include media(max-width $medium-screen) {
+    display: none;
+  }
+}
+
+#home-banner {
+  // background: image-url("home/banner-bg.jpg");
```
Comment (chenhoward) [inline app/assets/stylesheets/pages/home.scss]: Do we want to leave these comments in?

File: app/assets/stylesheets/projects/index.scss [hunk 3]
```diff
@@ -32,7 +33,7 @@ $demo-btn-shadow: #3C93D5;
 
 .demo-btn-color {
   background-color: $demo-btn-color;
-  box-shadow: 0 5px 0 0 $demo-btn-shadow;
+  // box-shadow: 0 5px 0 0 $demo-btn-shadow;
```
Comment (chenhoward) [inline app/assets/stylesheets/projects/index.scss]: Comments

File: app/assets/stylesheets/shared/tooltips.scss [hunk 4]
```diff
@@ -33,8 +33,9 @@
     @include transition (all 0.2s ease-in-out);
     background: $white;
     background-color: $tooltip-background;
+    border: 1px solid $gray-background-border-color;
     border-radius: $base-border-radius;
-    box-shadow: $tooltip-shadow;
+    // box-shadow: $tooltip-shadow;
```
Comment (chenhoward) [inline app/assets/stylesheets/shared/tooltips.scss]: Comments

File: app/views/projects/index.html.slim [hunk 5]
```diff
@@ -53,9 +53,9 @@
           h5.project-title
             | Project Homeless Connect
           p.project-description
-            | Provides a secure and paperless way for the homeless to sign up for services and for the organization to collect data to improve their service. Application built on Android with a Ruby on Rails API.
+            | Provides a secure way for the homeless to sign up for services and for volunteers to collect data. Application built on Android.
```
Comment (chenhoward) [inline app/views/projects/index.html.slim]: Buuuut the Rails part doooooe

Comment (kevintxwu) [inline app/views/projects/index.html.slim]: It's just so it ends up in 2 lines haha

File: config/locales/en.yml [hunk 6]
```diff
@@ -27,14 +27,14 @@ en:
...
-        Hey! We're Blueprint, a student organization using technology for social good.
-        We build web and mobile applications for nonprofits pro bono and foster the use of technology
-        for public welfare through events and discussions.
+        We're Blueprint, a student organization using technology for social good.
+        We build web and mobile apps for nonprofits pro bono and foster the use of technology
+        for public welfare through various events.
       our_projects: >
         Each year, teams of five students work with non-profit organizations on projects
-        to help them better serve their communities. We've built a crowdfunding application for
+        to help them better serve their communities. We have built a crowd-funding application for
```
Comment (chenhoward) [inline config/locales/en.yml]: I think grammatically we shouldn't even use 'have' but I'm no English major

File: app/views/projects/index.html.slim (hunk 5 above)
Comment (chenhoward) [inline app/views/projects/index.html.slim]: And we just append "and Rails"

Comment (kevintxwu) [inline app/views/projects/index.html.slim]: WORKS FOR ME!

**Comment by chenhoward** [comment]
Looking back at it I actually don't like the new laptop image on the home page and I like the old about page in terms of picture size/ lightness.  I would say having a lighter and larger picture gives more of the human aspect which is the aim of the about us page.  I also don't like the blue font, it's just not as easy to read as the black-grey-white, and it's randomly the one place on the website that's not within that color range.  I think titling the sections is redundant when the title is in the image.  I personally like our older nav bar because it takes up more space and I do like the old font better, but that's such  a small nit.  These are just my opinions on the new look but we should get some other opinions to comment on this.

**Pull request merged**